build.gradle

```diff
@@ -16,6 +16,9 @@ project.configurations {
 }
 
 project.dependencies {
+
+    implementation "org.xerial:sqlite-jdbc:3.7.2"
+    implementation("commons-dbcp:commons-dbcp:${commonsDbcpVersion}")
```

Comment:
See my comment in the other PR - I don't think this is necessary.

Reply:
Removed.